File: Android/java/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/dialog/SimpleDialogListener.java
package com.didichuxing.doraemonkit.widget.dialog;

import org.jetbrains.annotations.NotNull;

/**
 * Created by wanglikun on 2019/4/12
 */
public class SimpleDialogListener implements DialogListener {
    @Override
    public boolean onPositive(@NotNull DialogProvider<?> dialogProvider) {
        return false;
    }

    @Override
    public boolean onNegative(@NotNull DialogProvider<?> dialogProvider) {
        return false;
    }

    @Override
    public void onCancel(@NotNull DialogProvider<?> dialogProvider) {

    }

}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/widget/dialog/SimpleDialogListener.kt
package com.didichuxing.doraemonkit.widget.dialog

/**
 * Created by wanglikun on 2019/4/12
 */
open class SimpleDialogListener : DialogListener {
    override fun onPositive(): Boolean {
        return false
    }

    override fun onNegative(): Boolean {
        return false
    }

    override fun onCancel(): Boolean {
        return false
    }
}